cmd/machineset/main.go

```diff
@@ -36,6 +36,7 @@ func main() {
 	klog.InitFlags(nil)
 	watchNamespace := flag.String("namespace", "",
 		"Namespace that the controller watches to reconcile cluster-api objects. If unspecified, the controller watches for cluster-api objects across all namespaces.")
+	metricsAddress := flag.String("metrics-bind-address", ":8082", "Address for hosting metrics")
```

Comment:
Maybe we should use constants for the default ports for all the things that need this.  This way we don't miss something later.